sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -104,11 +105,34 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
                       // We should ignore this file from processing.
   )
 
+  private val configsAllJoinTypes = Seq(Seq(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key ->
+    SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.defaultValueString),
+    Seq(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1",
+      SQLConf.PREFER_SORTMERGEJOIN.key -> "true"),
+    Seq(SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1",
+      SQLConf.PREFER_SORTMERGEJOIN.key -> "false")) -> true
+
+  /**
+   * Maps a test with the set of configurations it has to run with and a flag indicating whether
+   * the output must be the same with different configs or it has to be different.
+   */
+  private val testConfigs: Map[String, (Seq[Seq[(String, String)]], Boolean)] = Map(
+    "typeCoercion/native/decimalArithmeticOperations.sql" ->
+      (Seq(Seq(SQLConf.DECIMAL_OPERATIONS_ALLOW_PREC_LOSS.key -> "true"),
+        Seq(SQLConf.DECIMAL_OPERATIONS_ALLOW_PREC_LOSS.key -> "false")) -> false),
+    "subquery/in-subquery/in-joins.sql" -> configsAllJoinTypes,
```

Comment:
This entry and the others blow don't change existing test result output?

Reply:
Exactly, changing these configs should not change the output.